core/unit_test/category_files/TestHIP_Category.hpp

```diff
@@ -49,6 +49,7 @@
 
 #define TEST_CATEGORY hip
 #define TEST_CATEGORY_NUMBER 6
+#define TEST_CATEGORY_DEATH hip_DeathTest
```

Comment:
It's also missing in HIPHostPinned

Reply:
I don;'t think the macro gets used for auxiliary spaces but sure I will add